dls-controls/malcolmjs
# Feature/new links #134

## Description

Added the ability to add new links, it optimistically updates the layout in the UI and then sends a PUT to the server to update the links on the associated attributes.

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- Run `npm start` and navigate to `http://localhost:300/gui/PANDA/layout`, move some blocks around and then make a new link.

## Agile board tracking

connect to #134
